libs/judicial-system/types/src/lib/notification.ts

```
@@ -62,12 +62,14 @@
~  CASE_FILES_UPDATED = CaseNotificationType.CASE_FILES_UPDATED,
~  DEFENDANT_SELECTED_DEFENDER = DefendantNotificationType.DEFENDANT_SELECTED_DEFENDER,
~  DEFENDER_ASSIGNED = DefendantNotificationType.DEFENDER_ASSIGNED,
~  INDICTMENT_SENT_TO_PRISON_ADMIN = DefendantNotificationType.INDICTMENT_SENT_TO_PRISON_ADMIN,
~  INDICTMENT_WITHDRAWN_FROM_PRISON_ADMIN = DefendantNotificationType.INDICTMENT_WITHDRAWN_FROM_PRISON_ADMIN,
~  SERVICE_SUCCESSFUL = SubpoenaNotificationType.SERVICE_SUCCESSFUL,
   SERVICE_FAILED = SubpoenaNotificationType.SERVICE_FAILED,
   SPOKESPERSON_ASSIGNED = CivilClaimantNotificationType.SPOKESPERSON_ASSIGNED,
   INDICTMENTS_WAITING_FOR_CONFIRMATION = InstitutionNotificationType.INDICTMENTS_WAITING_FOR_CONFIRMATION,
+  INDICTMENT_SENT_TO_PRISON_ADMIN = DefendantNotificationType.INDICTMENT_SENT_TO_PRISON_ADMIN,
```

Biome (1.9.4)

[error] Shouldn't redeclare 'INDICTMENT_SENT_TO_PRISON_ADMIN'. Consider to delete it or rename it.

'INDICTMENT_SENT_TO_PRISON_ADMIN' is defined here:


(lint/suspicious/noRedeclare)